app/javascript/controllers/index.js

```diff
@@ -1,6 +1,5 @@
 import { application } from "./application"
-import controllers from './**/*_controller.js'
-controllers.forEach((controller) => {
-  application.register(controller.name, controller.module.default)
-})
+import { definitionsFromContext } from "@hotwired/stimulus-webpack-helpers"
```

Comment:
I'd prefer import map over webpack, or any other js bundler that's not webpacker, webpacker gave us hard times in the past!